docs/mp/tracing/01_tracing.adoc

```diff
@@ -68,6 +115,718 @@ To disables MP Tracing as by specification:
 mp.opentracing.server.skip-pattern=.*
 ----
 
+==== Configuration using MP Config
+
+Tracing configuration can be defined in `microprofile-config.properties` file.
+
+[source, properties]
+.Tracing configuration
+----
+tracing.components.web-server.spans.0.name="HTTP Request"
+tracing.components.web-server.spans.0.logs.0.name="content-write"
+tracing.components.web-server.spans.0.logs.0.enabled=false
+----
+
+==== Path based configuration in Helidon Web Server
+
```

Comment:
It might be helpful to readers to group the config settings in a more task-oriented way. For example, have subsections for:
1. "Enabling and Disabling Tracing" which discusses disabling _all_ tracing, controlling by component, controlling by path, etc. 
2. "Controlling Tracing Output" (maybe there is a better name?) which describes renaming spans, changing the output format by span name, etc.
...and maybe more.

Reply:
I hope the proposed way corresponds to your idea :)

Reply:
The headings seem good. 

The "Controlling Tracing Output" section still contains some examples showing disabling by path and by component/spans. Those might fit better  the "Enabling and Disabling" section.